var/spack/repos/builtin/packages/libspng/package.py

```diff
@@ -14,10 +14,17 @@ class Libspng(CMakePackage):
     homepage = "https://github.com/randy408/libspng"
     url = "https://github.com/randy408/libspng/archive/refs/tags/v0.7.2.tar.gz"
 
+    maintainers("sethrj")
     license("BSD-2-Clause")
 
     version("0.7.4", sha256="47ec02be6c0a6323044600a9221b049f63e1953faf816903e7383d4dc4234487")
     version("0.7.2", sha256="4acf25571d31f540d0b7ee004f5461d68158e0a13182505376805da99f4ccc4e")
 
-    depends_on("c", type="build")  # generated
-    depends_on("cxx", type="build")  # generated
+    depends_on("c", type="build")
+    depends_on("zlib")
+
+    def cmake_args(self):
+        target = self.spec.target
+        args = [self.define("BUILD_EXAMPLES", False)]
+        args.append(self.define("ENABLE_OPT", target.vendor != "generic"))
+        return args
```

Comment:
```suggestion
    def cmake_args(self):
        target = self.spec.target
        return [ 
            self.define("BUILD_EXAMPLES", False)),
            self.define("ENABLE_OPT", target.vendor != "generic"),
        ]
```

Quick suggestion to clean up the syntax a bit.

Reply:
Yep thanks, I'd originally planned to put that second statement inside an `if` block (silly me).